src/locale/properties/record/record.en.properties

```diff
@@ -29,3 +29,7 @@ record.editThisRecord=Edit this record
 record.ogDescription=ORCID provides an identifier for individuals to use with their name as they engage in research, scholarship, and innovation activities.
 record.countryPublication=Country of publication
 record.description=Description
+record.noPublicInfo=No public information available.
+record.lockedRecord=This ORCID Record is locked
+thisAccountHasBeenDeprecated=This account has been deprecated, please see account
```

Comment:
could you replace the property and use the same format like `record.accountDeprecated` or something similar